collect_app/src/main/java/org/odk/collect/android/widgets/AnnotateWidget.java

```diff
@@ -63,7 +66,11 @@ public AnnotateWidget(Context context, QuestionDetails prompt) {
         imageCaptureHandler = new ImageCaptureHandler();
         setUpLayout();
         addCurrentImageToLayout();
-        addAnswerView(answerLayout);
+
+        Resources resources = context.getResources();
```

Comment:
It'd be good to pull this repeated code into `WidgetViewUtils`. Could be a method like `getStandardMargin()`.

**EDIT**: `WidgetViewUtils is actually not in the code yet as it is introduced in #3483. I think we should block this PR on that one as we may end up with conflicts. It should hopefully be merged soon 🤞.

Reply:
Yes, @seadowg, I also think that this bit of code is repetitive, and better should be specified in a separate class. It would be nice to have the PR merged with `WidgetViewUtils`, after that I can shift it there.